lang/csharp/.gitignore

```diff
@@ -1,35 +1,51 @@
-#   Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Please don't remove license headers (though it is odd that this .gitignore has one while the top level doesn't)

Were the changes to . gitignore automatically made? The lines about uncommenting bits under some circumstances makes me think so.

Reply:
1st - sorry for delay. I got dragged away with daily work so all my side-projects were on hold for some time.

I did not remove headers especially not license ones. I don't know what happened. 

Fixing that right now.